include/gauge_field_order.h

```diff
@@ -23,6 +23,7 @@
 #include <load_store.h>
 #include <aos.h>
 #include <transform_reduce.h>
+#include <communicator_quda.h>
```

Comment:
I need the number of ranks in all 4 directions (see `nproc` in the order class) to calculate offsets for the gauge, clover and spinor fields. Let me know if there is a better way to do that. in openQCD these are compile time constants.

Reply:
This line is removed.